scripts/get_version.sh

```diff
@@ -28,4 +28,4 @@
 
 set -e
 
-grep -oP "PROJECT_VERSION \"?\K[0-9.]+(?=\")?" "$(dirname "$0")/../CMakeLists.txt"
+sed -n -e 's/^\s*set(PROJECT_VERSION "\([0-9.]\+\)")\s*$/\1/p' "$(dirname "$0")/../CMakeLists.txt"
```

Comment:
This seem not to work on my mac. I played a bit with this and for me 
```
sed -n  's/^\s*set(PROJECT_VERSION "\(.*\)")$/\1/p'
```
 seem to work good.